.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
   ],
   // add your custom rules here
   rules: {
+    'import/namespace':         'off',
```

Comment:
> We previously kept a pristine .eslintrc.js copied from rancher/dashboard here

Yeah, that ship has long since sailed with upstream changes like this rancher/dashboard#8997. 

It definitely makes sense to keep this structured, but I'm wondering if we might be interested in publishing Dashboard's eslint rules instead of copying if we want to keep a similar base.